src/components/payments/splits/splitsColumns.tsx

```diff
@@ -0,0 +1,79 @@
+"use client";
+
+import { ColumnDef } from "@tanstack/react-table";
+import { Avatar, AvatarFallback, AvatarImage } from "@/components/ui/avatar";
+import { truncateString } from "@/app/(dashboard)/utils/stringUtils";
+import { SplitRecipient } from "./splitsRecipients";
+
+export const SplitsColumns: ColumnDef<SplitRecipient>[] = [
+  {
+    id: "avatar",
+    cell: ({ row }) => {
+      const reciepted = row.original;
+      const isArmitage =
+        reciepted.address == "0xB5685343eD45D8b896633F9c128C55F758feb0aA";
+      if (isArmitage) {
+        reciepted.username = "armitage-lab";
+      }
+      return (
+        <div>
+          {reciepted.username ? (
+            <Avatar className="h-8 w-8">
+              <AvatarImage
+                src={`https://github.com/${reciepted.username}.png?size=100`}
```

Comment:
Is the armitageLabs icon coming from this? the one with the weird checkmark

Maybe we also add that condition here that we talked for the armitage icon hardcoded

Reply:
was missing a "s" armitage-labs